.github/workflows/ci-pullrequest-main.yml

```diff
@@ -79,12 +79,44 @@ jobs:
   PackageDeploy:
     runs-on: ubuntu-latest
     needs: BuildTest
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
 
     steps:
     - uses: actions/checkout@v3
     
     - name: Docker Setup BuildX
       uses: docker/setup-buildx-action@v2
+
+    - name: Set image tag
+      run: |
+        echo "DOCKER_TAG_SHA=ghcr.io/buerokratt/mock-classifier:${GITHUB_SHA}" >> $GITHUB_ENV
+        echo "DOCKER_TAG_LATEST=ghcr.io/buerokratt/mock-classifier:latest" >> $GITHUB_ENV
```

Comment:
This variable is not used anywhere in the file, could it be removed?

Reply:
Good point. I'll fix that.